src/components/ENSListPage/ENSListPage.tsx

```diff
@@ -238,8 +310,95 @@ export default class ENSListPage extends React.PureComponent<Props, State> {
+        <div className="ens-page-header">
+          <div>
+            <h1>NAMEs</h1>
+            {`${ensList.length} results`}
+          </div>
+          <Button href={`${MARKETPLACE_WEB_URL}/names/claim`} target="_blank" primary>
+            Mint Name
+          </Button>
+        </div>
+        <Table basic="very">
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell width="2">{t('ens_list_page.table.name')}</Table.HeaderCell>
+              <Table.HeaderCell width="1">{t('ens_list_page.table.alias')}</Table.HeaderCell>
+              <Table.HeaderCell width="2">{t('ens_list_page.table.address')}</Table.HeaderCell>
+              <Table.HeaderCell width="2">{t('ens_list_page.table.land')}</Table.HeaderCell>
+              <Table.HeaderCell width="1">{t('ens_list_page.table.actions')}</Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            {paginatedItems.map((ens: ENS, index) => {
+              return (
+                <Table.Row className="TableRow" key={index}>
+                  <Table.Cell>{ens.subdomain}</Table.Cell>
+                  <Table.Cell>
+                    {this.isAlias(ens) ? (
+                      <span className="ens-list-owner">
+                        <Icon name="profile" />
+                        {t('ens_list_page.table.owner')}
+                      </span>
+                    ) : (
+                      <Button
+                        compact
+                        className="ens-list-btn"
+                        onClick={this.handleUseAsAlias.bind(null, ens.name)}
+                        disabled={!hasProfileCreated}
+                      >
+                        <Icon name="add" />
+                        {t('ens_list_page.button.add_to_avatar')}
+                      </Button>
+                    )}
+                  </Table.Cell>
+                  <Table.Cell>
+                    {ens.ensAddressRecord ? (
+                      <span className="ens-list-address">
+                        <img className="ens-list-address-icon" src={ethereumImg} alt="Ethereum" />
+                        {getCroppedAddress(ens.ensAddressRecord)}
+                        <CopyToClipboard role="button" text={ens.ensAddressRecord} showPopup={true}>
+                          <DCLIcon aria-label="Copy urn" aria-hidden="false" className="ens-address-copy" name="copy outline" />
+                        </CopyToClipboard>
+                      </span>
+                    ) : (
+                      <Button compact className="ens-list-btn" onClick={this.handleAssignENSAddress.bind(null, ens)}>
+                        <Icon name="add" />
+                        {t('ens_list_page.button.link_to_address')}
+                      </Button>
+                    )}
+                  </Table.Cell>
+                  <Table.Cell>{this.renderLandLinkInfo(ens)}</Table.Cell>
+                  <Table.Cell>TBD</Table.Cell>
```

Comment:
Are we missing something here?

Reply:
Yes! There is going to be a Name detail page and here there is an edit button. I will add it in another PR
<img width="254" alt="image" src="https://github.com/decentraland/builder/assets/11800206/cd58a193-ee5c-444b-a947-d8efba365f83">